charts/internal/cloud-provider-config/values.yaml

```diff
@@ -1,4 +1,3 @@
-kubernetesVersion: 1.17.0
```

Comment:
I guess https://github.com/gardener/gardener-extension-provider-azure/blob/5fb2542f9ec1adf7f1b0cf98582a647b2fb82e49/pkg/controller/controlplane/valuesprovider.go#L442 can be removed now.